File: Cloud Storage System/Level 1/C#/fileService.cs
// FileService.cs

public class FileService
{
    private readonly FileRepository _repository;

    public FileService(FileRepository repository)
    {
        _repository = repository;
    }

    public string AddFile(string path, int size)
    {
        var file = new File(path, size);
        return _repository.Add(file) ? "true" : "false";
    }

    public string GetFileSize(string path)
    {
        var file = _repository.Get(path);
        return file != null ? file.Size.ToString() : string.Empty;
    }

    public string DeleteFile(string path)
    {
        var file = _repository.Delete(path);
        return file != null ? file.Size.ToString() : string.Empty;
    }
}

File: Cloud Storage System/Level 1/python/file_service.py
# file_service.py

from file import File
from file_repository import FileRepository

class FileService:
    def __init__(self, repository: FileRepository):
        self.repo = repository

    def add_file(self, path: str, size: int) -> str:
        file = File(path, size)
        return "true" if self.repo.add(file) else "false"

    def get_file_size(self, path: str) -> str:
        file = self.repo.get(path)
        return str(file.size) if file else ""

    def delete_file(self, path: str) -> str:
        file = self.repo.delete(path)
        return str(file.size) if file else ""
